perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -577,10 +576,9 @@ sub can_report_missed {
 
     # Needs to be within 3 working days of the last completed round, today
     # not included.
-    # NOTE recent_collections shows the 'ideal'/expected collection date,
-    # not actual. So we need to check cab logs for actual collection date.
-    my $last_expected_collection_dt
-        = $property->{recent_collections}{ $service->{round_schedule} };
+    # NOTE last expected collection is the 'ideal'/expected collection date,
+    # not actual. So we need to check cab logs below for actual collection date.
+    my $last_expected_collection_dt = $service->{last} && $service->{last}{date};
```

Comment:
Don't know how to add a comment to a section of code that hasn't been changed, but the bit below with 
```
        # We need to check if the last collection was made over
        # WORKING_DAYS_WINDOW ago because some collections are weekly and some
        # fortnightly, but they share a round code prefix.
```
-- does that need changing?

Reply:
I don't _think_ so…

The bit below that you mention should now get a date for collections with multiple comma-separated round schedules, because this code is using `$service->{last}` which has done the processing for round schedules like `RES-NOR Fri, RES-NOR Tue`, whereas previously it was using `$property->{recent_collections}` directly.

Reply:
My brain is probably not clicking it together just right, but I'm still not sure it isn't a redundant check...BUT...worst case, it's just redundant. 👍 Will approve 🙂